src/SelectPanel/SelectPanel.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import {Meta} from '@storybook/react'
+import React, {useState} from 'react'
+import {theme, ThemeProvider} from '..'
+import {Button} from '../Button'
+import {TriangleDownIcon} from '@primer/octicons-react'
+import {ItemInput} from '../deprecated/ActionList/List'
+import BaseStyles from '../BaseStyles'
+import {SelectPanel} from '../SelectPanel'
+import Box from '../Box'
+
+const meta: Meta = {
+  title: 'Components/SelectPanel',
+  component: SelectPanel,
+  decorators: [
+    (Story: React.ComponentType<React.PropsWithChildren<unknown>>): JSX.Element => {
+      return (
+        <ThemeProvider theme={theme}>
+          <BaseStyles>
+            <Story />
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    },
+  ],
+  parameters: {
+    controls: {
+      disable: true,
+    },
+  },
+}
+export default meta
+
+function getColorCircle(color: string) {
+  return function () {
+    return (
+      <Box
+        bg={color}
+        borderColor={color}
+        width={14}
+        height={14}
+        borderRadius={10}
+        margin="auto"
+        borderWidth="1px"
+        borderStyle="solid"
```

Comment:
Could we please refactor these styles to use `sx` props instead of system props? [ref](https://primer.style/react/system-props)